### ISSUE TITLE
Question: bot comment burried in discussion

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [Levi-Lesches/blocking-issues#14]
Sometimes discussions get at a point where u have to press the load more button several times to see all comments made. Lets say mid discussion someone on the team decides that the PR needs to be blocked by another PR. More discussion continues and gets very long.

Is there anything we can do to make the comment from the bot always visible in some way? So u dont have to start looking for it in a lengthy discussion.

**Comment by Levi-Lesches** [comment]
I suppose I could modify the bot to add a footer to the original top comment instead. I remember having a reason for a specific comment from the bot but I can't remember it now. So, imagining this comment was the top comment for the issue, my content would end here and the bot would take over. 

----

# Status: Blocked :x:
### Issues blocking this PR: 
- #21 :heavy_check_mark:
- #22 :x:

This comment was automatically written by the [Blocking Issues](https://github.com/Levi-Lesches/blocking-issues) bot, and this PR will be monitored for further progress.
